Style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-image: url(Header-pic.png);
+  background-size: 1000px;
+  background-repeat: repeat-y;
+  background-repeat: repeat-x;
+}
+
+header {
+  background-color: aliceblue;
+  height: 40px;
+  display: flex;
+}
+
+.navbar {
+  font-size: large;
+}
+
+nav a {
+  margin: 10px;
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: larger;
+}
+
+.union {
+  width: 40px;
+  height: 40px;
+}
+
+li {
+  display: inline;
+  margin-left: 10px;
+}
+
+.content-container {
+  display: flex;
+  padding-top: 50px;
+  justify-content: center;
+  font-size: 25px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  flex-wrap: wrap;
+}
+
+.content {
+  margin-left: 35px;
+  margin-right: 35px;
+  text-align: center;
+}
+
+.name {
+  color: rgb(23, 11, 69);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+}
+
+.pusher {
+  margin: 45px;
+  text-align: center;
+}
+
+.job {
+  padding: 10px;
+  color: rgb(19, 168, 111);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+}
+
+.social-media {
+  text-align: center;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please follow the requirements of previous code reviewers who asked you to style the header page like [Figma's design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=8eIA2vsE9FRfB1Q2-0). I will try to explain it to you again. The background picture of your project should be fully reflected in the `mobile` version, and the `hx`, and `p` elements should be `positioned` as shown in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=8eIA2vsE9FRfB1Q2-0), the correct CSS properties should be given, and the `thickness` and `fonts` should also be as shown in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=8eIA2vsE9FRfB1Q2-0). should be Once again, I ask you to compare your project with the template in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=8eIA2vsE9FRfB1Q2-0) after completing this request to the end. We gave you tips for that you don't have any difficulties in the upcoming milestones, and we trust you to follow our advice. We wish you good luck 😃 


![image](https://user-images.githubusercontent.com/63915024/212129119-a776b5f4-c8ba-4234-84a7-a017b89888ee.png)